modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -316,6 +316,12 @@ TEST_P(Test_ONNX_layers, Resize)
     testONNXModels("resize_bilinear");
 }
 
+TEST_P(Test_ONNX_layers, ResizeUnfused)
+{
+    testONNXModels("upsample_unfused");
+    testONNXModels("resize_nearest_unfused");
+}
```

Comment:
Do you have test data for all the three subgraphs? Or just for two?

Reply:
I have test for only two i.e.` UpsampleSubgraph` and `ResizeSubgraph` and first one is getting tested in both tests. Actually these two are subgraph of upsample from differnet opset version (9 and 11). First (`ExtractScalesSubgraph`) is common in both, Actually `ExtractScalesSubgraph` will be part of each subgraph of upsample/Resize from any opset version.